### PR TITLE
Improve Unprotect Example

### PR DESCRIPTION
**Pull request by DyingAlbatross**
The prior example was more appropriate for demonstrating using protect on something that had been previously unprotected. This commit updates the example to use the newer syntax with .actions() and I think makes the use of unprotect a little more clear.

### CODE REVIEW

File: API.md [hunk 1]
```diff
@@ -947,17 +947,18 @@ In that case you can disable this protection by calling `unprotect` on the root
...
+    done: false
+}).actions(self => ({ 
     toggle() {
-        this.done = !this.done
+        self.done = !self.done
     }
-})
+}))
 
 const todo = new Todo()
```
Comment (mattiamanzati) [inline API.md]: Whops, this should be Todo.create() :P

**Comment by mattiamanzati** [comment]
Thanks for your PR! Unfortunately API.md is autogenerated from JSDocs in *.ts files, can you please move the changes over mst-operations.ts where the same comment exists? :)

File: API.md (hunk 1 above)
Comment (DyingAlbatross) [reply API.md]: Whoops is right! :)

**Comment by DyingAlbatross** [comment]
Sure thing!

**Comment by DyingAlbatross** [comment]
All set @mattiamanzati :thumbsup:

**Comment by mattiamanzati** [comment]
Awesome! Thanks for all of your work! :D

**Pull request merged**